.gitignore

```diff
@@ -35,7 +35,4 @@ bower_components
 Thumbs.db
 
 # Ignore built ts files
-compiledJSFiles/**/*
-
-# ignore yarn.lock
```

Comment:
Your project isn't using yarn, although if it was you would still want to commit this. It's like the `composer.json` of TS.